pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -36,6 +36,7 @@ import (
 
 const (
 	fetchAllPlugins = 1
+	openshiftEnabledAnnotation = "jenkins.dev/openshift-mode"
```

Comment:
`jenkins.dev/openshift-mode`, I think this is not the good name of annotation. Something like `jenkins.jenkins.io/openshift-enabled`. @tomaszsek @pawelprazak what do you think?

Reply:
How about just `jenkins.io/openshift-enabled` ?